### PR TITLE
Restructure the Ubuntu Getting Started section.

### PR DESCRIPTION
**Pull request by castrojo**
As per discussions on sig-cluster-lifecycle, we'd like our docs to be more lifecycle focused instead of one huge installation page. This breaks down the one big page into multiple smaller pages, it is mostly all new content.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1932)

### CODE REVIEW

**Comment by pwittrock** [comment]
@kubernetes/sig-cluster-lifecycle 

@roberthbailey Is this something should look at?

**Comment by marcoceppi** [comment]
:+1: expansive set of documentation for lifecycle of Kubernetes on Ubuntu

File: docs/getting-started-guides/ubuntu/backups.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+```
+results:
+  copy:
+    cmd: juju scp etcd/0:/home/ubuntu/etcd-snapshots/etcd-snapshot-2016-11-09-02.41.47.tar.gz
+      .
+  snapshot:
+    path: /home/ubuntu/etcd-snapshots/etcd-snapshot-2016-11-09-02.41.47.tar.gz
+    sha256: 1dea04627812397c51ee87e313433f3102f617a9cab1d1b79698323f6459953d
+    size: 68K
+status: completed
```
Comment (mikedanese) [inline docs/getting-started-guides/ubuntu/backups.md]: close this code block

File: docs/getting-started-guides/ubuntu/backups.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+## Restoring etcd data
+
+Allows the operator to restore the data from a cluster-data snapshot. This
+comes with caveats and a very specific path to restore a cluster:
+
+The cluster must be in a state of only having a single member. So it's best to
+deploy a new cluster using the etcd charm, without adding any additional units.
+
+    juju deploy etcd new-etcd
```
Comment (mikedanese) [inline docs/getting-started-guides/ubuntu/backups.md]: should this be in a code block?

File: docs/getting-started-guides/ubuntu/backups.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+> The above code snippet will deploy a single unit of etcd, as 'new-etcd'
+
+    juju run-action etcd/0 restore target=/mnt/etcd-backups
```
Comment (mikedanese) [inline docs/getting-started-guides/ubuntu/backups.md]: should this be in a code block?

File: docs/getting-started-guides/ubuntu/glossary.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# GLossary and Terminology
```
Comment (mikedanese) [inline docs/getting-started-guides/ubuntu/glossary.md]: GLossary

**Comment by castrojo** [comment]
Sorry, bad rebase, sorry for the spam folks, fixed.

**Comment by roberthbailey** [comment]
@castrojo - needs rebase

**Comment by roberthbailey** [comment]
Also assigning @jaredbhatti for tech writer review.

**Comment by mikedanese** [comment]
Tech LGTM

**Comment by castrojo** [comment]
Fixes #2041

**Comment by castrojo** [comment]
Fixes #1952

**Comment by castrojo** [comment]
- Fixes #1676 
- Fixes #1517 
- Fixes #1391 
- Fixes #703

**Comment by jaredbhatti** [comment]
Pinged to put content into our new templates: 
http://kubernetes.io/docs/contribute/page-templates/

Thanks!

**Comment by devin-donnelly** [comment]
@castrojo Docs LGTM. I think we can merge this.

However, we'll do so with the caveat that these kinds of guides to Bare Metal installation/running guides might/should move to a dedicated portion of the site.

File: docs/getting-started-guides/ubuntu/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+Title: Kubernetes on Ubuntu
+---
+
+{% capture overview %}
+There are multiple ways to run a Kubernetes with Ubuntu. These pages explain how to deploy Kubernetes on Ubuntu on multiple public and private clouds, as well as bare metal.
```
Comment (devin-donnelly) [inline docs/getting-started-guides/ubuntu/index.md]: "run a Kubernetes" -> "run Kubernetes" or "run a Kubernetes cluster"

Comment (castrojo) [inline docs/getting-started-guides/ubuntu/index.md]: Will be happy to move to a dedicated bare metal section once that's in place.

Comment (devin-donnelly) [inline docs/getting-started-guides/ubuntu/index.md]: I'm putting together a proposal that I hope to get out to sig-docs within a week.

**Comment by devin-donnelly** [comment]
Thanks! Merging now.

**Pull request merged**